Homework_1/Homework_1/Program.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+
+namespace Homework_1
+{
+    class Program
+    {
+
+        //Read name, surname, age, weight (in kg) and height(in cm) from console.
+        //Print a message to the screen based on the information given in 1. (an example message is shown below):
+        //Tom Jefferson is 19 years old, his weight is 50 kg and his height is 156.5 cm.
+        //Calculate and print body-mass index(BMI)
+        //Do 1 and 2 for another person.
+        static void Main(string[] args)
+        {
+            DisplayToConsole();
```

Comment:
Does the method name tell what it does? - I don't think so.
Rename the method so that it says what it does.